execute/source.go

```diff
@@ -26,11 +26,10 @@ type Source interface {
 }
 
 type CreateSource func(spec plan.ProcedureSpec, id DatasetID, ctx Administration) (Source, error)
-type CreateNewPlannerSource func(spec plan.ProcedureSpec, id DatasetID, ctx Administration) (Source, error)
 
-var procedureToSource = make(map[plan.ProcedureKind]CreateNewPlannerSource)
+var procedureToSource = make(map[plan.ProcedureKind]CreateSource)
```

Comment:
`CreateSource` and `CreateNewPlannerSource` define the same function signature, and `CreateSource` has no references. This seems to be a duplicated definition for me so I removed the longer one.

Same for `CreateTransformation` below.